plugin errors in Brackets 1.12

The extension manager displays "Extension error".

The following error is seen in the console:

```
/utils/ExtensionLoader.js:227 [Extension] failed to load C:/Users/Kiril/AppData/Roaming/Brackets/extensions/user/brackets-regex-diagram - Error: Module name "view/PanelManager" has not been loaded yet for context: _
http://requirejs.org/docs/errors.html#notloadederrback @ /utils/ExtensionLoader.js:227
/utils/ExtensionLoader.js:231 Error: Module name "view/PanelManager" has not been loaded yet for context: _
http://requirejs.org/docs/errors.html#notloaded
    at makeError (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:166:17)
    at Object.localRequire [as getModule] (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1400:44)
    at Object.<anonymous> (file:///C:/Users/Kiril/AppData/Roaming/Brackets/extensions/user/brackets-regex-diagram/main.js:10:39)
    at Object.context.execCb (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1658:33)
    at Object.Module.check (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:869:55)
    at Object.<anonymous> (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1121:34)
    at file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:132:23
    at file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1164:21
    at each (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:57:31)
    at Object.Module.emit (file:///C:/Program%20Files%20(x86)/Brackets/www/thirdparty/requirejs/require.js:1163:17)
```

The extension no longer does anything. This is as of the Brackets 1.12 release which I downloaded today.

--- a/main.js
+++ b/main.js
@@ -7,7 +7,6 @@ define(function (require, exports, module) {
     // get access to needed singletons
     var CommandManager     = brackets.getModule("command/CommandManager"),
         EditorManager      = brackets.getModule("editor/EditorManager"),
-        PanelManager       = brackets.getModule("view/PanelManager"),
         Menus              = brackets.getModule("command/Menus"),
         ExtensionUtils     = brackets.getModule("utils/ExtensionUtils"),
         PreferencesManager = brackets.getModule("preferences/PreferencesManager"),

